api/users/views.py

```diff
@@ -436,3 +443,66 @@ def perform_destroy(self, instance):
             if val['id'] in current_institutions:
                 user.remove_institution(val['id'])
         user.save()
+
+
+class UserAccountExport(JSONAPIBaseView, generics.CreateAPIView, UserMixin):
+    permission_classes = (
+        drf_permissions.IsAuthenticatedOrReadOnly,
+        base_permissions.TokenHasScope,
+        CurrentUser,
+    )
+
+    required_read_scopes = [CoreScopes.USERS_READ]
```

Comment:
Not at all sure myself what the answer is, but I remember in the [Node settings PR we worked on together](https://github.com/CenterForOpenScience/osf.io/pull/8536/files#diff-51eef6ea2a6c35e2b080c9a6fe591e38R1924) we decided to add a new core scope for settings, namely `NODE_SETTINGS_READ`. Is that a thing to be done here as well? My gut is that maybe `USER_SETTINGS` is different enough from just user read that it warrants its own scope, but that very well may be overkill!